.github/PULL_REQUEST_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,15 @@
+## Description
```

Comment:
Do we need this title?

Reply:
Why not...

Reply:
The field that we fill is already a description field. 😊 

I mean, check our current proper PR descriptions (like this), do you think it would be better if you add a title to your PR description as `## Description`?

Reply:
Makes sense.